### ISSUE TITLE
Stream point rewards don't work

### ISSUE THREAD

**Issue by Orange656** [oli414/OpenRCT2StreamIntegration#1]
I know this issue has already been addressed here. Only in-game notifications will appear. Some stream point rewards don't work. Tested also on older develop version of OpenRCT2 v0.3.3. Can you confirm? Thanks!
![obrazek](https://user-images.githubusercontent.com/37018328/196870727-878df92f-0b30-41a2-99c4-760a41991c1c.png)

**Comment by oli414** [comment]
Thanks for bringing this to my attention.

Are you using one of the default configurations?

**Comment by Orange656** [comment]
Yes, I looked at the config file and one of the few "Name The Park" works without a problem 👍

Renaming an attraction does not work for me if I'm typing this correctly ""old name" "new name""?

**Comment by oli414** [comment]
Not quite, please refer to this wiki page for the format of each action:
https://github.com/oli414/OpenRCT2StreamIntegration/wiki/Actions

**Comment by Orange656** [comment]
Ah! Well, that's my fault. I'll test it right away. But the weather can't be changed even according to the instructions :D
